src/js/api/config.js

```diff
@@ -163,9 +173,10 @@ const Config = function(options, persisted) {
 
     const parsedBwEstimate = parseFloat(config.bandwidthEstimate);
     const parsedBitrateSelection = parseFloat(config.bitrateSelection);
+    const parsedDefaultBwEstimate = parseFloat(config.defaultBandwidthEstimate);
     config.bandwidthEstimate = _isValidNumber(parsedBwEstimate) ? parsedBwEstimate : Defaults.bandwidthEstimate;
     config.bitrateSelection = _isValidNumber(parsedBitrateSelection) ? parsedBitrateSelection : Defaults.bitrateSelection;
-
+    config.defaultBandwidthEstimate = _adjustDefaultBwEstimate(parsedDefaultBwEstimate);
```

Comment:
`config.bandwidthEstimate ` should be set to `defaultBandwidthEstimate` when `null`. You can then close the commercial PR.